hardware/nic/generic.pan

```diff
@@ -0,0 +1,10 @@
+structure template hardware/nic/generic;
```

Comment:
@jouvin just added this in #117 as `hardware/nic/generic/generic_nic`, we don't need another one!